lib/puppet/type/sensu_client_subscription.rb

```diff
@@ -34,6 +34,11 @@ def initialize(*args)
     defaultto '/etc/sensu/conf.d/'
   end
 
+  newparam(:file_name) do
```

Comment:
This should be documented in the README

Reply:
Actually in README we document only sensu::subscription and from here we call sensu_client_subscription using the new file_name param (https://github.com/sensu/sensu-puppet/pull/846/commits/f8b25fe82584ebebab61461cd43e9803409ec531#diff-dbd03c1a865315be6893da851b471baeR29)  but not exposing it as param for sensu::subscription.
So either we expose file_name parameter also in sensu::subscription and we document it, or we just leave it a an internal setting that users don't have to mess it.

Reply:
ah ok, it can stay internal